### DISCUSSION TITLE
[Run issue]: General Question about Geant4-DNA physics constructors

### DISCUSSION THREAD

**Discussion by ShahinGHD** [OpenTOPAS/OpenTOPAS#59] [Run-Problem] [answered]
### Contact Details

shahin.ghasemi@gmail.com

### What was the issue?

Hi All, 
I have a question about using the GEANT4-DNA physics and its other options:
I have read the documentation and some papers describing different physics constructors (opt2, 4, and 6)

If one wants to use the DBSCAN algorithm to score the DNA breaks resulting from radionuclides, which of these physics modules is more relevant? g4em-dna or g4em-dna_opt6?

 Thanks
Shahin

### TOPAS category

Physics

### Version

4.0

### What Operative System (OS) are you running on?

Windows (WSL)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
This question was answered at https://github.com/topas-nbio/TOPAS-nBio/discussions/24
